AB_environments/AB_baseline.conda.yaml

```diff
@@ -5,7 +5,43 @@
 channels:
   - conda-forge
 dependencies:
+    # Copy-paste from recipe/meta.yaml, minus dask packages (see at bottom)
     - python=3.9
-    # Here we're using coiled-runtime for the sake of convenience, but we didn't have
-    # to. We could have also pointed to a nightly build.
-    - coiled-runtime=0.2.1
+    - pip
+    - coiled >=0.2.54
+    - numpy ==1.23.5
+    - pandas ==1.5.3
+    - fsspec ==2023.3.0
+    - s3fs ==2023.3.0
+    - gcsfs ==2023.3.0
+    - pyarrow ==11.0.0
+    - jupyterlab ==3.6.2
+    - dask-labextension ==6.1.0
+    - lz4 ==4.3.2
+    - ipywidgets ==8.0.4
+    - numba ==0.56.4
+    - scikit-learn ==1.2.2
+    - ipycytoscape ==1.3.3
+    - click ==8.1.3
+    - xarray ==2023.1.0
+    - zarr ==2.14.2
+    - msgpack-python ==1.0.5
+    - cloudpickle ==2.2.1
+    - tornado ==6.2
+    - toolz ==0.12.0
+    - zict ==2.2.0
+    - xgboost ==1.7.4
+    - dask-ml ==2023.3.24
+    - openssl >1.1.0g
+    - optuna ==3.1.0
+    - scipy ==1.10.1
+    - sqlalchemy ==1.4.46
+    - pynvml ==11.5.0
+    - bokeh ==2.4.3
```

Comment:
@jrbourbeau is bokeh 3 working now? I thought I saw a PR about it but can't find it at the moment.

Reply:
Yeah, `bokeh=3.1` is allowed in `distributed` `main` now -- though that change hasn't been released yet (xref https://github.com/dask/distributed/pull/7413)